eng/targets/RazorSDKDirectoryOverride.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+  <PropertyGroup>
+    <RazorSdkDirectoryRoot Condition="'$(RazorSdkDirectoryRootOverride)' != ''">$(RazorSdkDirectoryRootOverride)</RazorSdkDirectoryRoot>
+    <RazorSdkDirectoryRoot Condition="'$(RazorSdkDirectoryRootOverride)' == ''">$(ArtifactsBinDir)Microsoft.NET.Sdk.Razor\$(Configuration)\sdk-output\</RazorSdkDirectoryRoot>
```

Comment:
I think the general pattern would be:

```<RazorSdkDirectoryRoot Condition="'$(RazorSdkDirectoryRoot)' == ''">```

Reply:
Not quite, I want to be able to override the value with `RazorSdkDirectoryRootOverride` if it is set.

Reply:
I meant just for the second line - if `RazorSdkDirectoryRootOverride` is empty, `RazorSdkDirectoryRoot` will be empty on line 4. Using the name of the property you're setting indicates more that you're setting it's default value, if it doesn't already have a value, e.g.

https://github.com/dotnet/aspnetcore/blob/8c1bf1f1a31f852880c16575bca62507b75112bf/eng/targets/ResolveReferences.targets#L49-L57